.github/workflows/release-plz.yml

```diff
@@ -26,12 +26,34 @@ jobs:
       - name: Install Rust toolchain
         uses: dtolnay/rust-toolchain@stable
       - name: Run release-plz
+        id: release-plz
         uses: release-plz/action@v0.5
         with:
           command: release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+      
+      - name: Trigger Docker release workflow
+        if: ${{ steps.release-plz.conclusion == 'success' }}
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            console.log('Release-plz created releases. Triggering Docker release workflow...');
+            try {
+              const response = await github.rest.actions.createWorkflowDispatch({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                workflow_id: 'docker-release.yml',
+                ref: 'main'
```

Comment:
It seems like this is going to build whatever is in `main` and publish that as a docker image. Normally that would be fine, but it seems like it's possible that there would be a race where another change could merge before the release is published on DockerHub and then it doesn't match what was released on crates.io. Can you make it so when we call the docker-release workflow we somehow pass to it the release tag and make sure we build out of that branch for the docker image?